### ISSUE TITLE
ModuleNotFoundError after installing using pipx

### ISSUE THREAD

**Issue by DarwinAwardWinner** [flother/htmltab#6]
I followed the installation instructions using pipx:

```
pipx install git+https://github.com/flother/htmltab#egg=htmltab
```

It claims to have installed htmltab. However, when I run the command:

```
$ htmltab 
Traceback (most recent call last):
  File "/home/ryan/.local/bin/htmltab", line 5, in <module>
    from htmltab.cli import main
ModuleNotFoundError: No module named 'htmltab'
```

Looking in the virtualenv that pipx created, there is no htmltab module installed:

```
$ find ~/.local/pipx/venvs/htmltab/ -iname  '*htmltab*'
/home/ryan/.local/pipx/venvs/htmltab
/home/ryan/.local/pipx/venvs/htmltab/bin/htmltab
/home/ryan/.local/pipx/venvs/htmltab/lib/python3.9/site-packages/htmltab-0.1.1.dev4+g45b62c4.dist-info
```

**Comment by flother** [comment]
You're right, I see the same problem using Pipx 0.16.4 (installed via Homebrew):

```shell
$ pipx install git+https://github.com/flother/htmltab#egg=htmltab
  installed package htmltab 0.1.1.dev4+g45b62c4, Python 3.10.0
  These apps are now globally available
    - htmltab
done! ✨ 🌟 ✨
$ which htmltab
/Users/flother/.local/bin/htmltab
$ htmltab --version
Traceback (most recent call last):
  File "/Users/flother/.local/bin/htmltab", line 5, in <module>
    from htmltab.cli import main
ModuleNotFoundError: No module named 'htmltab'
```

I couldn't see any similar issues on other GitHub projects, so I'm not sure what the fix for this would be. I'll look into it further and see if I can isolate the problem.

For what it's worth, I can still install it using plain-old Pip:

```shell
$ pip install -e git+https://github.com/flother/htmltab#egg=htmltab
[...]
Successfully installed beautifulsoup4-4.10.0 cssselect-1.1.0 htmltab soupsieve-2.3.1
$ htmltab --version
htmltab, version 0.1.1.dev4+g45b62c4
```

**Comment by flother** [comment]
Well, it's commit 9637a032b2fc1fdb34d21da803ddda4a70b2f008 that's broken the installation process. If I install from the previous commit, 4f1fa0b2ea1a79cd64ad206865c93eeb07096ac2, then I can install it successfully:

```shell
$ pipx install git+https://github.com/flother/htmltab.git@4f1fa0b2ea1a79cd64ad206865c93eeb07096ac2
  installed package htmltab 0.1.0rc0, Python 3.10.0
  These apps are now globally available
    - htmltab
done! ✨ 🌟 ✨
$ htmltab --version
htmltab, version 0.1.0rc0
```

I moved to [a `src` layout](https://hynek.me/articles/testing-packaging/) in the commit that broke things. Perhaps I missed something important in the process. I'll fix that as soon as I can.

**Comment by flother** [comment]
It was a simple fix in the end: I added an empty `src/htmltab/__init__.py` file to convert the package from a [namespace package](https://packaging.python.org/en/latest/guides/packaging-namespace-packages/) into a standard package (see a68fb1dd). After fixing it I found pypa/pipx#162, where a comment implies Pipx doesn't (and shouldn't) support installing a namespace package. I'm not sure why that's the case though.

Thanks for letting me know about the problem, I appreciate you taking the time to submit the issue.
